frontend/src/pages/profile/libs/helpers/get-activity-breakpoint/get-activity-breakpoint.helper.ts

```diff
@@ -0,0 +1,30 @@
+import { ACTIVITY_BREAKPOINTS } from '../../constants/constants.js';
+
+const getActivityBreakpoint = (activityCount: number): string => {
+  let activityBreakpoint = '';
+  const { breakpoint1, breakpoint2, breakpoint3, breakpoint4 } =
+    ACTIVITY_BREAKPOINTS;
+
+  switch (true) {
+    case activityCount === 0: {
+      activityBreakpoint = breakpoint1;
+      break;
+    }
+    case activityCount === 1: {
+      activityBreakpoint = breakpoint2;
+      break;
+    }
+    case activityCount > 1 && activityCount < 5: {
+      activityBreakpoint = breakpoint3;
+      break;
+    }
+    case activityCount >= 5: {
+      activityBreakpoint = breakpoint4;
+      break;
+    }
+  }
+
+  return activityBreakpoint;
```

Comment:
As now it's in separate helper - we can simplify it
And again `B31`
```suggestion
  const { breakpoint1, breakpoint2, breakpoint3, breakpoint4 } =
    ACTIVITY_BREAKPOINTS;

  switch (true) {
    case activityCount === 0: {
      return breakpoint1;
    }
    case activityCount === 1: {
      return breakpoint2;
    }
    case activityCount > 1 && activityCount < 5: {
      return breakpoint3;
    }
    case activityCount >= 5: {
      return breakpoint4;
    }
    default: {
      return '';
    }
  }
```